apps/frontend/src/components/App.tsx

```diff
@@ -12,7 +12,7 @@ import {
   Toolbar,
   Typography,
 } from "@mui/material";
-import { Route, Routes, useNavigate } from "react-router-dom";
+import { Route, Routes, Link as RouterLink, Link } from "react-router-dom";
```

Comment:
Here your importing `Link` twice. Once as `RouterLink` and once as `Link`. Did you intended `Link` to be imported from MUI?